assayist/processor/utils.py

```diff
@@ -195,8 +195,8 @@ def get_source_of_build(build_info):
 
     for value in task_request:
         # Check if the value in the task_request is a git URL
-        if isinstance(value, str) and re.match(r'git\+?(https|ssh)?://', value):
-            return value
+        if isinstance(value, str) and re.match(r'git\+?(http[s]?|ssh)?://', value):
+            return re.sub(r'^git\+http', r'http', value)
```

Comment:
I guess to keep this DRY, we should turn this into a utility function (or decorator) and run it on all returned values?

Reply:
Or we can call it after `get_source_of_build` in `download_source`. What do you prefer?

Reply:
After, in `download_source`.

Reply:
@sdherr, 
Does it still apply if we consider #79?

IMHO, it is fine just in `download_source` because we do not use URL by any utility in main analyzer.

Reply:
We don't care what the url is in the analyzers, just that there is one so we can record it in the db. I would actually prefer to keep the url we record to the db identical to what is in brew.